maestro-cli/src/main/java/maestro/cli/command/TestCommand.kt

```diff
@@ -158,6 +158,19 @@ class TestCommand : Callable<Int> {
     )
     private var headless: Boolean = false
 
+    @Option(
+        names = ["--analyze"],
+        description = ["[Beta] Enhance the test output analysis with AI Insights"],
+    )
+    private var analyze: Boolean = false
+
+    @Option(names = ["--apiUrl"], description = ["[Beta] API base URL"])
+    private var apiUrl: String = "https://api.copilot.mobile.dev"
+
+
+    @Option(names = ["--apiKey"], description = ["[Beta] API key"])
```

Comment:
don't we have this already? `--api-key`?

Reply:
Not in this command, neither in parent command like `--verbose`